### PR TITLE
Viite 2420 fix for node form save button

### PR DESCRIPTION
**Pull request by andre-policarpo**

### CODE REVIEW

File: viite-UI/src/less/core/forms.less [hunk 1]
```diff
@@ -473,14 +487,15 @@ form.node .control-label-small {
...
 
-input.junction-number-input {
-  text-align: center;
-  width: 20px !important;
-  margin-right: 11px;
-  &:invalid, &:invalid:focus {
-    border-color: @error;
-  }
-}
+//input.junction-number-input {
```
Comment (Campanha) [inline viite-UI/src/less/core/forms.less:490]: commented code should be deleted

**Comment by peltsi212** [comment]
I think word hotfix should be reserved for updates to production that are done outside of normal publishing cycle. This save button problem has not been in production so this is not hotfix.

**Comment by andre-policarpo** [comment]
> I think word hotfix should be reserved for updates to production that are done outside of normal publishing cycle. This save button problem has not been in production so this is not hotfix.

I actually didn't know how/when the word `hotfix` should be used. Thanks for the knowledge. I'll keep that in mind for future fixes and hotfixes.

File: viite-UI/src/less/core/forms.less (hunk 1 above)
Comment (andre-policarpo) [reply viite-UI/src/less/core/forms.less]: Done

**Pull request merged**